playground/src/pages/CodingEnvironment.tsx

```diff
@@ -15,6 +16,98 @@ async function getExample(example: string) {
 
 function CodingEnvironment() {
 
+   /**
+  * Helper function to check if input contains only numbers.
+  */ 
```

Comment:
please fix the indentation. IMO we should have [prettier](https://prettier.io/docs/en/install.html) or something like it in the CI / pre-commit hooks.

Reply:
Indeed. I already have it, but when I let it run now, the changes in this PR grow very big. 

I suggest I will run prettier after this merge in a second PR? But happy to do otherwise

Reply:
In that case I would just manually fix this and open the follow up later on

Reply:
got it